microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+[ICoreWebView2_16::ShowPrintUI(printDialogKind) Method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+[COREWEBVIEW2_PRINT_DIALOG_KIND Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_dialog_kind)
+
+
+---
+
+<!-- end of tab-set -->
+
+### Example: ShowPrintUI
+
+This example shows the user a print dialog. If `printDialog` is `CoreWebView2PrintDialogKind.Browser`, opens a browser print preview dialog, `CoreWebView2PrintDialogKind.System` opens a system print dialog.
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+```csharp
+void ShowPrintUI(object target, ExecutedRoutedEventArgs e)
+{
+  string printDialog = e.Parameter.ToString();
+  if (printDialog == "Browser")
+  {
+    // Opens the browser print preview dialog.
+    webView.CoreWebView2.ShowPrintUI();
+  }
+  else
+  {
+    // Opens the system print dialog.
+    webView.CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind.System);
+  }
+}
+```
+
+**APIs:**
+
+* [CoreWebView2.ShowPrintUI() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui)
+* [CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind))
+* [CoreWebView2PrintDialogKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printdialogkind)
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+```cpp
+bool AppWindow::ShowPrintUI(COREWEBVIEW2_PRINT_DIALOG_KIND printDialogKind)
+{
+  wil::com_ptr<ICoreWebView2_16> webView2_16;
+  CHECK_FAILURE(m_webView->QueryInterface(IID_PPV_ARGS(&webView2_16)));
+  CHECK_FAILURE(webView2_16->ShowPrintUI(printDialogKind));
+  return true;
+}
+```
+
+**APIs:**
+
+* [ICoreWebView2_16::ShowPrintUI(printDialogKind) Method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+* [COREWEBVIEW2_PRINT_DIALOG_KIND Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_dialog_kind)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## Print
+
+The Print method silently print the current top-level document in the WebView2 using optional programmatically specified print settings. If you want to build your own print preview dialog or otherwise build your own print experience you can use this method. This API consists of an asynchronous Print method and a PrintSettings object.
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+[CoreWebView2.PrintAsync() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
```

Comment:
```suggestion
* [CoreWebView2.PrintAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
```
style convention for these regular docs: omit param list / parentheses from link text, except in very rare case of overload - and then, use param name (init lowercase), not the verbose type.